wherehows-web/app/components/dataset-compliance.ts

```diff
@@ -820,11 +824,12 @@ export default class DatasetCompliance extends Component {
         [TagFilter.showAll]: identity,
```

Comment:
Consider changing to new computed style :)

Reply:
new computed style?

Reply:
this is an object using the strategy pattern and computed property names to resolve to a function
```js
func = {
  [computedA]: () => A,
  [computedB]: () => B
}[valueAorB]
```
Using property getters does not really provide an enhancement for this use case and seems cumbersome to index in ts